feature-libs/user/profile/components/update-email/update-email-component.service.ts

```diff
@@ -17,7 +18,8 @@ export class UpdateEmailComponentService {
     protected userEmail: UserEmailFacade,
     protected routingService: RoutingService,
     protected globalMessageService: GlobalMessageService,
-    protected authService: AuthService
+    protected authService: AuthService,
+    protected authRedirectService: AuthRedirectService
```

Comment:
Let's add 4.0 migration:
- add bullet in the 4_0.md migration docs that this service requires new dependency AuthRedirectService
- add schematics (that add the new constructor dependency)